cmd/all-in-one/Dockerfile

```diff
@@ -1,8 +1,14 @@
-ARG base_image
-ARG debug_image
+ARG TARGET=release
 
-FROM $base_image AS release
-ARG TARGETARCH=amd64
+FROM golang:1.15-alpine AS base-image-debug
+
+FROM alpine:3.12 AS cert
+RUN apk add --update --no-cache ca-certificates
+
+FROM scratch AS base-image-release
```

Comment:
Are we changing back the base image back to scratch ?

Reply:
Good catch! No, we shouldn't change it back to scratch.

Reply:
i asked. 

Why we need to change to alpine for the release image

Reply:
See #2545 for context.

Reply:
Sure. 
got info from https://github.com/jaegertracing/jaeger/issues/2116. 
I have put it back.